week1/homework/src/server.js

```diff
@@ -9,12 +9,33 @@ function createServer(port) {
   let state = 10;
 
   const server = http.createServer((request, response) => {
-    // TODO: Write your homework code here
+    response.writeHead(200, { 'Content-Type': 'application/json' });
```

Comment:
I would say that we should not send a response status code of 200 until we know what we have hit a valid path.